doc/user/pimv6.rst

```diff
@@ -0,0 +1,248 @@
+.. _pim:
```

Comment:
Don't you see a warning here? I think we should use a different _tag_ like pim6 or so. Otherwise, how could you reference actually this document?